package.json

```diff
@@ -38,5 +38,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@babel/plugin-proposal-private-methods": "^7.18.6"
```

Comment:
why did we need this package?

Reply:
When I created a new branch left-navigation it didn't work so I searched for the error. They said need this package. Should I uninstall this package?